ops/services/virtual_network/main.tf

```diff
@@ -20,18 +20,6 @@ resource "azurerm_virtual_network" "vn" {
   tags = var.management_tags
 }
 
-# VMs subnet
-resource "azurerm_subnet" "vms" {
```

Comment:
Does Terraform cleanly destroy this and other VM resources, or do we have to clean up any manually created resources (network connections?, IPs, etc) before we can run this against an environment?

Reply:
Posting the answer here for posterity, since we talked about this in a standup: Yes, we will have to clean up manually-created resources before this can be run. I will take care of that across all environments as part of the deployment window mentioned above.